docs/guides/tutorials/rest_apis_with_fastapi.rst

```diff
@@ -0,0 +1,535 @@
+.. _ref_guide_rest_apis_with_fastapi:
+
+===============================
+REST APIs with EdgeDB & FastAPI
+===============================
+
+EdgeDB can help you quickly build REST APIs in Python without getting into the rigmarole of using ORM libraries to handle your data effectively. Here, we'll be using `FastAPI <https://fastapi.tiangolo.com/>`_ to expose the API endpoints and EdgeDB to store the content.
+
+We'll build a simple event management system where you'll be able to fetch, create, update, and delete *event hosts* and *events* via RESTful API endpoints.
+
+Prerequisites
+=============
+
+Before we start, make sure you've :ref:`installed <ref_admin_install>` EdgeDB and EdgeDB-CLI. In this tutorial, we'll use Python 3.10 and take advantage of the asynchronous I/O paradigm to communicate with the database more efficiently. A working version of this tutorial can be found `here <https://github.com/edgedb/edgedb-examples/tree/main/fastapi-crud>`_ on GitHub.
+
+
+Install the dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To follow along, clone the repository and head over to the ``fastapi-crud`` directory. Create a Python 3.10 virtual environment, activate it, and install the dependencies with this command:
+
+.. code-block:: bash
+    pip install edgedb fastapi httpx[cli] uvicorn
+
+
+Initialize the database
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Now, let's create a new EdgeDB instance for this project. From the root directory, run:
+
+.. code-block:: bash
+    edgedb project init
+
+You should see the following prompts on your console:
+
+::
+    Initializing project...
+
+    Specify the name of EdgeDB instance to use with this project
+    [default: fastapi_crud]:
+    > fastapi_crud
+
+    Do you want to start instance automatically on login? [y/n]
+    > n
+    Checking EdgeDB versions...
+
+Once you've answered the prompts, a new EdgeDB instance called ``fastapi_crud`` will be created. Start the server with the following command:
+
+.. code-block:: bash
+    edgedb instance start fastapi_crud
```

Comment:
We're about to change `edgedb project init` to always keep the instance running (even when you answered "n" to "Do you want to start instance automatically on login? [y/n]")  (https://github.com/edgedb/edgedb-cli/issues/710)

cc @tailhook @colinhacks @elprans 

But maybe for the sake of this tutorial and to avoid complications *now* we just suggest to answer "y" and then we can avoid this dance with 'edgedb instance start'?

Reply:
Sure. I think `y` should be the answer. If we don't think that auto-start instance is the most common use case we should just drop this question.

Reply:
I thought the instance keeps running in the background—just like docker containers—when someone logs out. If not, then auto initialization is something that I'd want.